cloudigrade/account/serializers.py

```diff
@@ -89,6 +89,20 @@ def create(self, validated_data):
             instances_data = aws.get_running_instances(session)
             with transaction.atomic():
                 account.save()
+                try:
+                    aws.configure_cloudtrail(session, aws_account_id)
+                except ClientError as error:
+                    if error.response.get('Error', {}).get('Code') == \
+                            'AccessDeniedException':
+                        raise serializers.ValidationError(
+                            detail={
+                                'account_arn': [
```

Comment:
"Permission denied for ARN foo" is what we use in the case that we can't any session for the role at all. Since at this point we _can_ get the session, we should have something else unique to this use case. Maybe just reword this like "Access denied to create CloudTrail for ARN foo."